.evergreen/deploy.yml

```diff
@@ -33,71 +36,86 @@ functions:
           AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
           AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
           AWS_SESSION_TOKEN: ${AWS_SESSION_TOKEN}
-        script: |
-          ${PREPARE_SHELL}
-          # Add PROFILER to env if profiler patch parameter is set to true.
-          if [ "${profiler}" = "true" ]; then
-              echo "PROFILER is set to true, adding profiler to the build"
-              PROFILER=true PROFILE_HEAD="<script src=\"https://localhost:8097\"></script>" yarn env-cmd --no-override -e "${target}" yarn deploy-utils-build-and-push
-          else
-              yarn env-cmd --no-override -e "${target}" yarn deploy-utils-build-and-push
-          fi
+        binary: bash
+        args:
+          - -c
+          - |
+            ${PREPARE_SHELL}
+            # Add PROFILER to env if profiler patch parameter is set to true.
+            if [ "${profiler}" = "true" ]; then
+                echo "PROFILER is set to true, adding profiler to the build"
+                PROFILER=true PROFILE_HEAD="<script src=\"https://localhost:8097\"></script>" yarn env-cmd --no-override -e "${target}" yarn deploy-utils-build-and-push
+            else
+                yarn env-cmd --no-override -e "${target}" yarn deploy-utils-build-and-push
+            fi
   email:
-    - command: shell.exec
+    - command: subprocess.exec
       params:
         working_dir: ui/${app_dir}
         env:
           EVERGREEN_API_SERVER_HOST: ${evergreen_api_server_host}
           EVERGREEN_UI_SERVER_HOST: ${evergreen_ui_server_host}
           EVERGREEN_API_KEY: ${evergreen_api_key}
           EVERGREEN_USER: ${evergreen_user}
-        script: |
-          echo "populating evergreen.yml"
-          cat <<EOF > .evergreen.yml
-          api_server_host: $EVERGREEN_API_SERVER_HOST
-          ui_server_host: $EVERGREEN_UI_SERVER_HOST
-          api_key: $EVERGREEN_API_KEY
-          user: $EVERGREEN_USER
-          EOF
-          echo "Done populating evergreen.yml"
-    - command: shell.exec
+        binary: bash
```

Comment:
For some of the longer scripts, it might be nice to create script files that we can run. This is also recommended by our [docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Best-Practices#subprocessexec)